packages/dart/sshnoports/README.md

```diff
@@ -4,13 +4,12 @@
 [![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/atsign-foundation/noports/badge)](https://api.securityscorecards.dev/projects/github.com/atsign-foundation/noports)
 [![SLSA 3](https://slsa.dev/images/gh-badge-level3.svg)](https://slsa.dev)
 
-# SSH! No ports
+# NoPorts Cli Binaries
 
-ssh no ports provides a way to ssh to a remote linux host/device without that
-device or the client having any open ports (not even 22) on external interfaces. All
-network connectivity is out bound and there is no need to know the IP
-address the device has been given. As long as the device and client has an IP address (public or private 1918),
-DNS and Internet access, you will be able to connect to it.
+This is the source code for the NoPorts cli binaries.
```

Comment:
s/cli/Command Line Interface (CLI)/

Reply:
fixed in [0e23e54](https://github.com/atsign-foundation/noports/pull/1613/commits/0e23e54562176433076875067b0a3f7c94563856)